### PR TITLE
MDTOCRenderer: increase compatibility with python-markdown

### PR DESCRIPTION
**Pull request by gotmax23**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -24,11 +24,13 @@ markdown_extensions:
   - toc:
       permalink: true
   # pymdownx
+  - pymdownx.escapeall
   - pymdownx.highlight:
       anchor_linenums: true
   - pymdownx.superfences
 nav:
   - index.md
+  - changelog.md
```
Comment (felixfontein) [inline mkdocs.yml:33]: I would probably move this to the end of the list and adjust the title to "Changelog of antsibull-changelog". WDYT?

**Review by felixfontein — COMMENTED** [review]
Thanks, this looks great!

File: mkdocs.yml (hunk 1 above)
Comment (felixfontein) [reply mkdocs.yml:33]: ping @gotmax23

Comment (gotmax23) [reply mkdocs.yml]: See the latest push

**Comment by gotmax23** [comment]
I enabled preview builds from readthedocs in this repo. Let's see if it works...

File: mkdocs.yml [hunk 2]
```diff
@@ -24,11 +24,13 @@ markdown_extensions:
...
+  - pymdownx.escapeall
   - pymdownx.highlight:
       anchor_linenums: true
   - pymdownx.superfences
 nav:
   - index.md
+  - antsibull-changelog Release Notes: changelog.md
   - changelogs.md
   - other-projects.md
   - changelog-configuration.md
```
Comment (felixfontein) [inline mkdocs.yml:36]:
```suggestion
  - changelogs.md
  - other-projects.md
  - changelog-configuration.md
  - antsibull-changelog Release Notes: changelog.md
```

File: mkdocs.yml [hunk 3]
```diff
@@ -32,4 +33,5 @@ nav:
...
+  - antsibull-changelog Release Notes: changelog.md
   - changelog.yaml-format.md
```
Comment (felixfontein) [inline mkdocs.yml:37]:
```suggestion
  - changelog.yaml-format.md
  - antsibull-changelog Release Notes: changelog.md
```

**Review by felixfontein — APPROVED** [review]

**Comment by gotmax23** [comment]
Thanks for the review!

**Pull request merged**